server/views/media_picker.py

```diff
@@ -148,66 +154,72 @@ def _cached_featured_collection_list(tag_id_list):
 
 # helper for preview queries
 # tags_id is either a string or a list, which is handled in either case by the len() test. ALL_MEDIA is the exception
-def concatenate_query_for_solr(solr_seed_query, media_ids, tags_ids, custom_ids=None):
-    custom_ids = [] if custom_ids is None else custom_ids
+
+def concatenate_query_for_solr(solr_seed_query, media_ids, tags_ids, custom_collection=None):
```

Comment:
I'm not sure if you've rebased against the latest, but the value for `custom_collections` should be set to a default array below too. Something like what's done in https://github.com/mitmedialab/MediaCloud-Web-Tools/blob/443a6c8609c2879d51c722ae647f673a6267cfd1/server/views/media_picker.py#L152.

Reply:
got it. pylinter gave me an 'dangerous' error assigning the custom_collections=[] as default.